addons/web/static/src/legacy/js/widgets/data_export.js

```diff
@@ -397,10 +397,14 @@ var DataExport = Dialog.extend({
             self._onShowData(records);
             self.$('.o_fields_list').empty();
 
+            let forceDefaultExportFields = _.map(_.filter(records, r => r.default_export), r => r.id);
```

Comment:
I know that most of the code uses underscore, but since the new code is using ES6 syntax (variable declarations & lambdas) it really has no reason to use underscore's HoFs.

Also `const`.

Reply:
And have you checked with the JS team if they're not rewriting this after the owl 2 merge or something? Because it would be a bit of a shame if that was merged in 15.3 then deleted in 15.4 because it was missed / forgotten in the conversion.

Reply:
Thanks, indeed, I have changed, it is better using ES6. I will ask the JS team Monday.